src/Microsoft/Silk.NET.DirectComposition/Structs/FRAMESTATISTICS.gen.cs

```diff
@@ -0,0 +1,84 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+using System.Text;
+using Silk.NET.Core;
+using Silk.NET.Core.Native;
+using Silk.NET.Core.Attributes;
+using Silk.NET.Core.Contexts;
+using Silk.NET.Core.Loader;
+
+#pragma warning disable 1591
+
+namespace Silk.NET.DirectComposition
+{
+    [NativeName("AnonymousName", "__AnonymousRecord_dcomptypes_L78_C9")]
+    [NativeName("Name", "DCOMPOSITION_FRAME_STATISTICS")]
+    public unsafe partial struct FRAMESTATISTICS
```

Comment:
@Perksey do you know what happened here? this is very odd...

Reply:
There are a few cases like this and I don't know why they happen, really we need to add another path in the namer to detect when the "friendly name" doesn't look friendly (i.e. is all caps). I think Humanizer can convert nicely from SCREAMING_SNAKE_CASE to PascalCase, but obviously at the moment we don't actually tell Humanizer what source case we have (so it has to make its best guess, which seems to be wrong)